glean-core/ios/Glean/Config/Configuration.swift

```diff
@@ -9,7 +9,6 @@ public struct Configuration {
     let userAgent: String
     var logPings: Bool
     let maxEvents: Int32?
-    var pingTag: String?
```

Comment:
Would you kindly file a bug for removing this from the C# bindings too _after_ we have environment variables ready?=

Reply:
Sorry, didn't think to check the C# bindings here. Filed the bug https://bugzilla.mozilla.org/show_bug.cgi?id=1648007